docs/time/index.rst

```diff
@@ -1013,9 +1013,10 @@ To get the representation of a |Time| object::
   >>> import matplotlib.pyplot as plt  # doctest: +SKIP
   >>> jyear = np.linspace(2000, 2001, 20)  # doctest: +SKIP
   >>> t = Time(jyear, format='jyear')  # doctest: +SKIP
-  >>> plt.plot_date(t.plot_date, jyear)  # doctest: +SKIP
-  >>> plt.gcf().autofmt_xdate()  # orient date labels at a slant  # doctest: +SKIP
-  >>> plt.draw()  # doctest: +SKIP
+  >>> fig, ax = plt.subplots()  # doctest: +SKIP
```

Comment:
I don't understand why everything is skipped and it is not even a plot directive, so no nice plot is being added to the doc. Maybe a `time` maintainer should review.

Reply:
Okay, thanks for your feedback. I opened https://github.com/astropy/astropy/issues/17454 as follow-up issue.